src/lib/style/base/_core.scss

```
@@ -1057,22 +1058,22 @@ a.dropdown-item {
   }
 }
 .SRC-success-color {
-  color: #28a745;
+  color: map.get(SRC.$colors, 'success');
 }
 .SRC-success-color-bg {
-  background-color: #28a745;
+  background-color: map.get(SRC.$colors, 'success');
 }
 .SRC-warning-color {
-  color: #e0b603;
+  color: SRC.$yellow-warning;
```

Any reason to not do this just like the 'success' in stylesheet?
```suggestion
  color: map.get(SRC.$colors, 'warning');
```